quizzes/09_arrow.js

```diff
@@ -0,0 +1,160 @@
+console.log(
+  // curryAdd(2)(4)(6),
+  // genSeries(),
+  // genSeries(7, 100),
+  // shift("havin pasta for brunch"),
+  // shift("havin pasta for brunch", 13),
+  // shift("univa cnfgn sbe oehapu", 13),
+  // getPopulation().display()
+)
+
+function curryAdd(a) {
+  // TODO: refactor this function
+  // rewrite with arrow syntax
+  return function(b) { // eslint-disable-line func-names
+    return function(c) { // eslint-disable-line func-names
+      return a + b + c
+    }
+  }
+}
+
+function genSeries(n = 10, limit = 10) {
+  // TODO: refactor this function
+  // remove the loop
+  // Use spread opertaor, Array functions in ES6, like Array.from()
+
+  // this function returns an array of n random integers
+  // Each integer can be less than or equal to limit
+  // limit is 10 by default, if not provided
+  const arr = []
+
+  for (let i = 0; i < n; i++) {
+    arr.push(Math.floor(Math.random() * limit))
+  }
+  return arr
+}
+
+
+ // This is a complex example,
+ // mostly to show that arrow refactoring
+ // doesn't need to hurt readability
+ // You may use () => () or () => {}
+ // based on how complex you think the code can be
+function shift(str, amount = 1) {
```

Comment:
Oh no! This happens every time I let ESLint fix things automatically - it converts functions to arrow functions.

But you are correct - there isn't much difference between the code and refactored code.